README.md

```diff
@@ -48,11 +49,11 @@ Backend Repository for LandVille Project
 | GET    | api/v1/properties/enquiries/all/                   | get all your enquiries                                   |
 | GET    | api/v1/properties/enquiries/<enquiry_id>/          | get one enquiry                                          |
 | DELETE | api/v1/properties/:slug/resource                   | delete a cloudinary resource                             |
-| GET    | api/v1/auth/profile/                               | gets a profile of the loggedin user
+| GET    | api/v1/auth/profile/                               | gets a profile of the loggedin user                      |
 
-| PATCH  | api/v1/auth/profile/                               | edits a profile of the loggedin user
+| PATCH | api/v1/auth/profile/ | edits a profile of the loggedin user
```

Comment:
I know this is not supposed to be part of your PR, but could you please add pipes to the ends of these lines so that they are properly tabulated ...

Reply:
Noted @MandelaK , will add it